docs/DeveloperGuide.md

```diff
@@ -2,14 +2,219 @@
 
 ## Acknowledgements
 
-{list here sources of all reused/adapted ideas, code, documentation, and third-party libraries -- include links to the original source as well}
+{list here sources of all reused/adapted ideas, code, documentation, and third-party libraries -- include links to the
+original source as well}
 
 ## Design & implementation
 
 {Describe the design and implementation of the product. Use UML diagrams and short code snippets where applicable.}
 
+### [Implemented] Add Account
```

Comment:
is there a reason why [Implemented] was used for already implemented features instead of [Pending] for un-implemented features like in the course website?